intel-retail/retail-use-cases
# refactor: update compose and Makefile to use retail root env variable…

… instead of hard coding for volume mounts

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**issue_number**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

git clone this PR
make run docker compose for all use cases
ensure that the env variable changes have no impact on the make targets

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/retail-use-cases )
